Structure changes

GKnolPicturae opened the pull request:
Added order fields and small UI changes for "Deed"/"Persoonsvermelding" records and added a new record type named "Bezoekersregistratie"

reinvantveer commented:
@GKnolPicturae thanks! Unfortunately the checks are failing for this PR but there's a couple of simple things to do. The `hidden` property isn't part of the publicly available Memorix ontology yet, but you can set an entry in the `config.yaml` to ignore the URI for this. Either that, or update the Memorix ontology publication ;P

GKnolPicturae commented:
Ah. So that's what the fail was. 
Thanks. I did both. I added the missing property to the config file and I also added the new Shacl to the "todo" list.
